setup.py

```diff
@@ -131,9 +129,10 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
 
 setup(
     name='django-ses',
-    version='0.8.14',  # When changing this, remember to change it in __init__.py
+    version='0.9.0',  # When changing this, remember to change it in __init__.py
     packages=find_packages(exclude=['example', 'tests']),
     package_data=package_data,
+    python_requires='>=3.5.8',
```

Comment:
```suggestion
```

Reply:
thanks so much for your input! I changed the python version to `3.4.10` and removed python 2.7 in the travix matrix. After all, python 2.7 is dead. Long live python 3.

Reply:
I'm fine dropping support for python 2, though if it's not a big deal we could keep it (I suspect some users will be affected). However it's not worth a lot of time... And hopefully not too many users complain.

It would be great if you could add the supported Django versions to the README. As far as that, I'd be fine supporting 1.10 or even 1.11 and above. Django 3 should be easy if 2.2 works...

Reply:
Ok, when the tests are fixed, I'll give Python 2.7 another shot and I added django 1.10 to the tox and travis.